docker-compose.yml

```diff
@@ -17,5 +19,15 @@ services:
       - ./frontend:/app
       - /app/node_modules
     ports:
-      - "3000:3000"
+      - "4000:4000"
       - "24678:24678"
+
+  db:
+    image: postgres:15
+    container_name: postgres_db
+    environment:
+      - POSTGRES_DB=${DATABASE_NAME}
+      - POSTGRES_USER=${DATABASE_USER}
+      - POSTGRES_PASSWORD=${DATABASE_PASSWORD}
+    ports:
+      - "5432:${DATABASE_PORT}"
```

Comment:
Should `DATABASE_PORT` rather be `5432`? Or was there a reason to choose `3306`?

Reply:
Actually this correct will make the changes